_articles/offices-are-expensive.md

```diff
@@ -26,7 +26,7 @@ Early in the pandemic, before there was any prospect of the office reopening, it
 
 And this is before you even considered the personal costs of the commute and the lack of flexibility for team members.
 
-Suffice it to say, we didn't catastrophically fail at remote working. Our work continued at at least he pace that it had before, and our team continued to grow, excel and have fun.
+Suffice it to say, we didn't catastrophically fail at remote working. Our work continued at, at least the pace that it had before, and our team continued to grow, excel and have fun.
```

Comment:
Thanks for the typo fix. I don't think putting a comment there is correct though.

Reply:
I could change it to `at least at the pace` I suppose, maybe that's a bit less clunky.